interfaces/vault_kv/v0/schema.py

```diff
@@ -35,6 +37,14 @@ class AppVaultKvRequirerSchema(BaseModel):
     mount_suffix: str = Field(
         description="Suffix to append to the mount name to get the KV mount."
     )
+    wrap_ttl: int = Field(
```

Comment:
@ghislainbourgeois Should this be restricted to just `int`, can we use `int` or `null`? I'm not familiar with the intricacies of charm relations here, but it seems like we should still accept `null` to make it optional.

@skatsaounis I know we pushed `int`-only on you, but I just want to clarify we were really trying to say "drop the `string` part".

Reply:
Hopefully the last version can resolve this thread. Please take a look :slightly_smiling_face:

Reply:
Hopefully, except this was a question I was posing to @ghislainbourgeois  :laughing: -- I don't necessarily know this is the approach we want to take.